handlers/common.go

```diff
@@ -42,7 +42,7 @@ func GetValidatorOnlineThresholdSlot() uint64 {
 // GetValidatorEarnings will return the earnings (last day, week, month and total) of selected validators
 func GetValidatorEarnings(validators []uint64, currency string) (*types.ValidatorEarnings, map[uint64]*types.Validator, error) {
```

Comment:
Aside from the validator page this function is only also used for the dashboard.
Please also check that these changes don't break anything.